docs/current/sdk/python/628797-get-started.md

```diff
@@ -37,6 +37,12 @@ The code samples in this tutorial are based on the above FastAPI project. If usi
 The Dagger Python SDK requires [Python 3.10 or later](https://docs.python.org/3/using/index.html). Using a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) is recommended.
 :::
 
+Create a virtual environment for your project (if you don't already have one):
+
+```shell
+python3 -m venv .venv && source .venv/bin/activate
+```
+
```

Comment:
Right above this, there's this note: **Using a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) is recommended.**

That link takes you to instructions on how to set it up, problem is that users may be glancing over the note.

We should combine both of these. Perhaps something like this:

```suggestion
The Dagger Python SDK requires [Python 3.10 or later](https://docs.python.org/3/using/index.html). 
:::

Create a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) for your project (if you don't already have one). Example:

```shell
python3 -m venv .venv && source .venv/bin/activate
```